ios/PurchasesHybridCommon/PurchasesHybridCommonTests/StoreProductDiscount+HybridAdditionsTests.swift

```diff
@@ -59,7 +59,7 @@ public extension SKProductDiscount {
     }
 }
 
-private extension SKProductSubscriptionPeriod {
+public extension SKProductSubscriptionPeriod {
```

Comment:
Maybe make it internal?
```suggestion
extension SKProductSubscriptionPeriod {
```